tests/wind_data_integration_test.py

```diff
@@ -1109,3 +1109,69 @@ def test_read_csv_long_ti():
 
     expected_result = np.array([0.06, 0.07])
     np.testing.assert_allclose(wind_ti_rose.turbulence_intensities, expected_result)
+
+
+def test_wind_rose_nan_values():
```

Comment:
These tests don't seem to actually test the desired behavior. They pass on both the develop branch (without the new NaN check) and with the new code.

Reply:
I've now fixed this by adding a `match` kwarg to `pytest.raises()`